File: Producto/routes-service/src/main/java/com/tagok/routes_service/dto/response/route/CobroPorticoResponse.java
package com.tagok.routes_service.dto.response.route;

import java.math.BigDecimal;
import java.time.LocalDateTime;

import com.tagok.routes_service.domain.tarifa.TipoTarifa;

public record CobroPorticoResponse(
    Long porticoId,
    String nombre,
    String codigo,
    String autopista,
    double latitud,
    double longitud,
    TipoTarifa tarifa,
    BigDecimal valor,
    LocalDateTime fechaHora) implements CobroRutaResponse
{
}

File: Producto/tag-ok-app/app/src/main/java/com/tagok/app/data/dto/route/CobroPorticoResponse.kt
package com.tagok.app.data.dto.route

import kotlinx.datetime.LocalDateTime
import kotlinx.serialization.Serializable

@Serializable
data class CobroPorticoResponse(
    val porticoId: Long?,
    val nombre: String?,
    val codigo: String?,
    val autopista: String,
    val latitud: Double,
    val longitud: Double,
    val tarifa: String,
    val valor: Double,
    val fechaHora: LocalDateTime) : CobroRutaResponse()
